<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>文泉驿微米黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>数据结构课程设计</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>10</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数据结构课程设计</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="subtitleLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>53</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
    </font>
   </property>
   <property name="text">
    <string>——用户登录系统的模拟</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>355</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="infoHorizontalLayout">
    <item>
     <widget class="QLabel" name="AuthorLabel">
      <property name="text">
       <string>软工1603 杨振华 201626811225</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="infoHorizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="DateLabel">
      <property name="text">
       <string>2017.11</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="filenameEditor">
        <property name="font">
         <font>
          <family>Code New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>data.txt</string>
        </property>
        <property name="placeholderText">
         <string>文件名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="loadDataButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>加载数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveDataButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>保存数据</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="text">
         <string>重置AVL树视图</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="usernameEditor">
        <property name="font">
         <font>
          <family>Code New Roman</family>
          <italic>false</italic>
         </font>
        </property>
        <property name="placeholderText">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="passwordEditor">
        <property name="font">
         <font>
          <family>Code New Roman</family>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
        <property name="placeholderText">
         <string>密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="loginButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>用户登录</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userModifyButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>密码修改</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userAddButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>用户添加</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="userDeleteButton">
        <property name="font">
         <font>
          <family>文泉驿微米黑</family>
         </font>
        </property>
        <property name="text">
         <string>用户删除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="operationStatusLabel">
      <property name="font">
       <font>
        <family>文泉驿微米黑</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>操作状态</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
